docs/accessControl.md

```diff
@@ -66,13 +66,13 @@ Every course belongs to an **institution** and by default access is only allowed
 
 Each user accesses the PrairieLearn server in a `mode`, as listed below. This can be used to restrict access to assessments based on the current mode. The `mode` setting of an access rule has the following effect:
 
-| Mode        | When is access allowed?                                                                                                                                                                                                          |
-| ----------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `Public`    | Access is normally allowed, but is blocked when the student is signed in for an exam in the Computer-Based Testing Facility (CBTF).                                                                                              |
-| `Exam`      | Access is normally blocked, and is only allowed when the student is signed in to the Computer-Based Testing Facility (CBTF). The `examUuid` should also be specified in the same access rule to limit access to a specific exam. |
-| no mode set | Access is allowed anytime, both during CBTF exams and outside of exams.                                                                                                                                                          |
+| Mode        | When is access allowed?                                                                                                                                                                                                                                                               |
+| ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `Public`    | Access is normally allowed, but is blocked when the student is checked in on PrairieTest, either in a Computer-Based Testing Facility (CBTF) or in a course-managed session.                                                                                                          |
+| `Exam`      | Access is normally blocked, and is only allowed when the student is checked in on PrairieTest, either in a Computer-Based Testing Facility (CBTF) on in a course-managed session. The `examUuid` should also be specified in the same access rule to limit access to a specific exam. |
```

Comment:
In PT we always use the phrase "testing center" so I think it might be better to never use the phrase CBTF and instead write something like:
```suggestion
| `Exam`      | Access is normally blocked, and is only allowed when the student is checked in on PrairieTest, either in a testing center or in a course-managed session. The `examUuid` should also be specified in the same access rule to limit access to a specific exam. |
```

Reply:
Done in a5aedf4c8 in multiple locations. I had to be careful not to say "centre", let me know if you see a slip... :grin: